src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingsService.java

```diff
@@ -168,6 +159,25 @@ private void getPublicAndPrivateData(String entityNumber, String passThroughToke
     privateDataRetrievalService.initialisePrivateData(entityNumber);
   }
 
+  private void setUpdateSubmissionData(Map<String, Object> data,
```

Comment:
This will be updated as part of [UAR-494](https://companieshouse.atlassian.net/browse/UAR-494), in the meantime it might be an idea to change this method just to return the UpdateSubmission model object as described in my [other comment](https://github.com/companieshouse/overseas-entities-api/pull/238#discussion_r1201735649)

Reply:
I changed things when I merged.